feature-detects/css/hsla.js

```diff
@@ -6,10 +6,10 @@
   "tags": ["css"]
 }
 !*/
-define(['Modernizr', 'createElement', 'contains'], function(Modernizr, createElement, contains) {
+define(['Modernizr', 'setCss', 'createElement', 'contains'], function(Modernizr, setCss, createElement, contains) {
   Modernizr.addTest('hsla', function() {
-    var style = createElement('a').style;
-    style.cssText = 'background-color:hsla(120,40%,100%,.5)';
-    return contains(style.backgroundColor, 'rgba') || contains(style.backgroundColor, 'hsla');
+    var elem = createElement('a');
+    setCss(elem, 'background-color:hsla(120,40%,100%,.5)');
+    return contains(elem.style.backgroundColor, 'rgba') || contains(elem.style.backgroundColor, 'hsla');
```

Comment:
```suggestion
    return contains(style.backgroundColor, 'rgba') || contains(style.backgroundColor, 'hsla');
```

from copied PR - I had an error here - but maybe I'm wrong